app/src/main/res/layout/fragment_sidebar_main.xml

```diff
@@ -240,7 +240,7 @@
             <TextView
                     android:id="@+id/version_info"
                     android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
+                    android:layout_height="?attr/listPreferredItemHeight"
```

Comment:
This text is rendered at the top of its boundary.

![slice 2](https://cloud.githubusercontent.com/assets/11763113/22022958/8371569c-dd07-11e6-9c4e-ae7450cc5d4d.png)

I think it is better to set also `gravity=center`